data/decks/phonon.json

```diff
@@ -0,0 +1,216 @@
+{
+	"id": "phonon",
+	"season": 6,
+	"display_name": "Phonon",
+	"cardback": "cardback_phonon.jpg",
+	"exceed_cost": 1,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "strike"
+		}
+	}],
+	"character_action_exceeded": [
+		{
+			"gauge_cost": 1,
+			"force_cost": 0,
+			"effect": {
+				"condition": "last_turn_was_strike",
+				"effect_type": "choice",
+				"choice": [
+					{ "effect_type": "close", "amount": 1, "and": { "effect_type": "strike" } },
+					{ "effect_type": "retreat", "amount": 1, "and": { "effect_type": "strike" } }
+				],
+				"negative_condition_effect": {
```

Comment:
I think this is backwards (close/retreat if this is *not* the turn after a strike)

Reply:
> oh there's a not_last_turn_was_strike condition, can just use that instead of moving everything to the negative

Reply:
actually as brought up in tech support it seems to let you close or retreat regardless of whether the last turn was a strike? not sure why that would be the case though

Reply:
fixed